tests/usbus_cdc_acm_stdio/Makefile

```diff
@@ -0,0 +1,23 @@
+BOARD ?= samr21-xpro
+include ../Makefile.tests_common
+
+USEMODULE += auto_init_usbus
+USEMODULE += usbus_cdc_acm
```

Comment:
This is pulled in as a dependency of `stdio_cdc_acm`

Reply:
You're right, removed.